python/amici/ode_export.py

```diff
@@ -1494,8 +1496,20 @@ def _compute_equation(self, name: str) -> None:
             self._eqs[name] = sp.zeros(self.ncl(), self.nx_solver())
 
         elif name == 'dtcldp':
-            # force symbols
-            self._eqs[name] = self.sym(name)
+            # check, whether the CL consists of only one state. Then,
+            # sensitivities drop out, syms can be deleted
+            state_set = {state.get_id() for state in self._states}
+            self._eqs[name] = sp.Matrix([
+                [0] * self.np()
+                if len(state_set.intersection(
+                    tcl.get_val().free_symbols)) == 1
+                else [
+                    sp.Symbol(f's{strip_pysb(tcl.get_id())}__'
```

Comment:
this should ideally reuse the symbols from the syms matrix, prone to diverge like that. Alternatively implement a function that generates symbol names and use it in both places.

Reply:
As discussed below, will do.